src/Components/Profile.js

```diff
@@ -25,8 +25,12 @@ function Profile({ profile, username }) {
             className="p-mr-2 p-mt-2"
           />
         </Avatar>
-        <h1 className="p-m-2">{name}</h1>
-        <h4 className="">({username})</h4>
+        <div className="p-m-2 p-d-flex">
+          <h1 className="p-m-0 p-r-3">{name}</h1>
+          <div className="p-m-2 p-d-flex">
+            <h1 className="p-m-0 p-r-3">{name}</h1>
```

Comment:
There should only ever be one h1 per page due to accessibility reasons. So we either need an h1 and a h2 both name and username should be inside a h1 with an extra span to add different styles to the username.

Reply:
How's this coming along? Do you need any help @SPARKY892 ?